zadani/soft/style.css

```diff
@@ -11,33 +13,166 @@
 body {
     margin: 0;
     padding: 0;
-    background-color: #f5f5f5;
     font-family: 'Merriweather', serif;
     font-size: 16px;
     line-height: 1.6;
+    background-color: #f5f5f5;
 }
 
-
 /* HLAVICKA */
 
+.hlavicka {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    background-image: url('images/hlavicka.jpg');
```

Comment:
Protože složka _images_ není ve stejném adresáři, jako tenhle soubor _style.css_. Musíš jít ob 2 složky nahoru, takže `'../../images/hlavicka.jpg'` by bylo správně 🙂

Reply:
K obrázku na pozadí by to pak chtělo přidat další vlastnosti, např. aby se vměstnal do celé velikosti hlavičky (`background-size: cover`), atd.